trk/src/NfcUtils/readClimb.js

```diff
@@ -2,25 +2,42 @@ import NfcManager from 'react-native-nfc-manager';
 
 async function readClimb() {
   let climbIdBytes = [];
+  let gradeBytes = [];
+  let colorBytes = [];
```

Comment:
Climbs aren't always named after 'colors'. They can be called anything (e.g., bubblegum, The Crag, 3301).

What about 'nameBytes'? Or any other suggestions?